discover/portal/articles/03-collaboration/02-collaborating/08-informing-users-with-the-knowledge-base.markdown

```diff
@@ -1109,7 +699,10 @@ you can display one or more *sections* (groups) of articles. Navigate to the
 setting to add a new field for each section you want, and enter a name for each 
 section:
 
-*Admin KBArticle sections*(Business, Politics or World for example)
+*Admin KBArticle sections*
+Business
+Politics
+World
```

Comment:
This flat list of section names does not correlate to how you're explaining that the user uses the plus sign in the UI. Instead, it would be better to show an image of those section names entered in separate fields in the UI.

Note, as they're currently formatted, all four of these lines show as a single paragraph.